tools/pydocgen/pydocgen/__init__.py

```diff
@@ -106,7 +106,8 @@ def read_input(input_file: str, package_override: str = "") -> Input:
         providers = []
         if package_override != "":
             provider = [obj for obj in input_dict.get("providers") if obj["package_name"] == package_override]
-            providers.append(Provider(**provider[0]))
+            if len(provider) > 0:
```

Comment:
Nit: I believe it's more "pythonic" to check for non-empty like:

```suggestion
            if provider:
```

Reply:
Ha! I was literally reading a thread about that, but felt that checking for a list's length would be more explicit about what we want this to be. But I don't regularly write Python, so I don't know what is preferred. I can change it per your suggestion if you still think I should?